README.md

```diff
@@ -30,8 +30,12 @@ OJP is part of the [Transmodel](https://www.transmodel-cen.eu/) family of standa
 * [OJP 1.0.1](https://github.com/VDVde/OJP/releases/tag/v1.0.1): Bugfix release
 * [OJP 1.0](https://github.com/VDVde/OJP/releases/tag/v1.0): The version described in CEN/TS 17118:2017
 
+## Documentation
+We provide [documentation tables](https://vdvde.github.io/OJP/index.html) which have been generated from the XSD.
+
 ## GitHub Repository
-Download the XSD schema files and examples in the git repository found at https://github.com/VDVde/OJP and take a look at the [documentation tables](https://vdvde.github.io/OJP/index.html).
+Download the XSD schema files and examples in the git repository found at [https://github.com/VDVde/OJP](https://github.com/VDVde/OJP). 
```

Comment:
```suggestion
Download the XSD schema files and examples from the git repository found at [https://github.com/VDVde/OJP](https://github.com/VDVde/OJP). 
```

Reply:
I imagined that we could introduce it just for the future and leave it for 2.0, thus having this small difference. But the problem is really minor, I have absolutely no problem if you prefer merging it just like that.